python/ray/serve/master.py

```diff
@@ -653,6 +656,19 @@ async def update_backend_config(self, backend_tag, config_options):
             await self._start_pending_replicas()
             await self._stop_pending_replicas()
 
+            await self.broadcast_backend_config(backend_tag)
+
+    async def broadcast_backend_config(self, backend_tag):
+        _, backend_config, _ = self.backends[backend_tag]
+        for replica_tag in self.replicas[backend_tag]:
+            try:
+                replica = ray.util.get_actor(replica_tag)
+            except ValueError:
+                continue
+            # We don't await here so it's just fire and forget.
```

Comment:
I think we need to await here. Otherwise, what if the replica is currently being restarted and the master actor crashes after making these calls and updating its checkpoint?